### DISCUSSION TITLE
级联选择器直接引用antd组件，下拉无数据；从@formily/antd中引用样式错乱

### DISCUSSION THREAD

**Discussion by tangtang09** [alibaba/formily#1707] [Q&A 2.x] [answered]
**级联选择器**
1、引用antd组件，option已配置，但是下拉显示无数据，如下：
[https://codesandbox.io/s/interesting-thunder-qzdhu?file=/App.tsx](url)

2、引用@formily/antd组件，option已配置过，但点击下拉无反应，发现是层级不够被覆盖，修改样式强制让下拉显示时，样式错乱（项目全局查找，并未全局修改过），这个现象无法用codesandbox重现，以下是项目中的错乱图

<img width="908" alt="tta" src="https://user-images.githubusercontent.com/83819365/124251711-cf82e880-db58-11eb-8538-78db2b639e54.png">

**Comment by janryWang** [comment, marked as answer]
直接从antd依赖的Cascader，因为没有做dataSource映射，所以肯定拿不到数据，需要从@formily/antd里引的，看看这个 https://codesandbox.io/s/cool-wiles-gsrx6?file=/App.tsx

**Reply by tangtang09** [reply]
引用@formily/antd的，codesandbox确实可以正常运行，但是在我本地项目中，同样的写法无法正常运行，就如上图中所示

**Reply by janryWang** [reply]
确认版本是不是最新的

**Reply by tangtang09** [reply]
都是最新版本哦
<img width="342" alt="Snipaste_2021-07-05_11-22-03" src="https://user-images.githubusercontent.com/83819365/124425779-a2724800-dd9b-11eb-8065-4a59b30ea188.png">

**Comment by jeeeet** [comment]
我也遇到了, 引用@formily/antd, 下拉选择框样式都是错的, 下拉选择项也不显示, 但页面上有数据, 请问这个问题解决了吗, 还是哪里用法不对
